tailnet/coordinator.go

```diff
@@ -194,6 +200,8 @@ func (c *coordinator) AgentCount() int {
 // ServeClient accepts a WebSocket connection that wants to connect to an agent
 // with the specified ID.
 func (c *coordinator) ServeClient(conn net.Conn, id uuid.UUID, agent uuid.UUID) error {
+	logger := c.logger.With(slog.F("client_id", id), slog.F("agent_id", agent))
+	logger.Debug(context.TODO(), "coordinating client")
```

Comment:
Please add a note for why you're using `context.TODO()` here, it's unclear for the reader what's to be done.

(Alternatively, consider switching to `Background()`.)

Reply:
I was hedging a bit when I wrote this because it seemed like maybe we do want a context for these functions.

Thinking about it a bit more, the lifecycle is tied to the `conn`, so I'll just use `context.Background()`